client/src/components/Onboarding/OnboardingUser.module.scss

```diff
@@ -1,23 +1,15 @@
-@import url('../../styles/settings/space.css');
-
-.root {
-  padding: var(--global-space--section);
-}
-
 .access {
   text-align: right;
 }
 
-.title {
-  width: 100%;
-  border-bottom: 1px solid black;
-  margin-bottom: 1em;
+.content {
+  padding-bottom: 1em;
```

Comment:
**Why?** Ensure that scrolling (enabled via this commit) does not leave button touching bottom of content pane.